pkg/apis/serving/v1alpha1/llm_inference_service_types.go

```diff
@@ -126,10 +126,13 @@ type LLMModelSpec struct {
 	// +optional
 	Name *string `json:"name,omitempty"`
 
-	// Criticality defines how important it is to serve the model compared to other models.
-	// This is used by the Inference Gateway scheduler.
+	// Priority defines how important it is to serve this model compared to other models.
+	// This is used by the Inference Gateway scheduler for request prioritization.
+	// Higher values indicate higher priority. Negative values are allowed.
+	// If unset, defaults to 0. This maps to the Priority field in InferenceObjective.
+	// For example: 10 (high priority), 0 (default), -10 (low priority/sheddable).
 	// +optional
-	Criticality *igwapi.Criticality `json:"criticality,omitempty"`
+	Priority *int `json:"priority,omitempty"`
```

Comment:
since this is a breaking change, to be able to upgrade objects from one version to another, we'll have to create a v1alpha2 API and add a conversion webhook for v1alpha1 <-> v1alpha2